tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -25,7 +24,7 @@
 NUMBER_OF_INSTANCES = int(os.environ.get("RANCHER_AIRGAP_INSTANCE_COUNT", "1"))
 IMAGE_LIST = os.environ.get("RANCHER_IMAGE_LIST", ",".join(
     [TEST_IMAGE, TEST_IMAGE_NGINX, TEST_IMAGE_OS_BASE])).split(",")
-ARCH = os.environ.get("RANCHER_ARCH", "amd64")
+TARBALL_TYPE = os.environ.get("RANCHER_TARBALL_TYPE", "tar.gz")
```

Comment:
Does the `RANCHER_` prefix really fit? Would it be better to change to `K3S_` or `RKE2_` or some other common prefix?

Reply:
We have historically used the `RANCHER_` prefix for k3s and rke2 things. I do think it would be good to update to K3S/RKE2, so I'm making that change for the variables here, but we'll need to refactor quite a lot of code or just make sure future k3s/rke2 automation uses appropriately named variables.